packages/graphback-cli/src/utils/generateHelper.ts

```diff
@@ -0,0 +1,71 @@
+import chalk from 'chalk';
+import { accessSync, readFileSync, writeFileSync } from 'fs';
+import { GlobSync } from 'glob'
+import { GraphQLBackendCreator, IGraphQLBackend, KnexResolverManager, PostgresSchemaManager } from 'graphback-core'
+import { logError, logInfo } from '.';
+
+/**
+ * Message after command execution
+ */
+function postCommandMessage(): void {
+  logInfo(`
+Successfully generated schema and resolvers :tada:.  
+
+Next steps:
+1. Run ${chalk.cyan(`graphback db`)} to create postgres database.
+2. Run ${chalk.cyan(`docker-compose up -d`)} followed by ${chalk.cyan(`npm start`)}
+`)
+}
+
+/**
+ * Generate schema and resolvers using graphback-core and
+ * write them into generated folder
+ */
+async function generateBackend(): Promise<void> {
+  try{
+    accessSync(`${process.cwd()}/model`)
+    const model = new GlobSync('model/*.graphql', { cwd: process.cwd()})
+    
+    if(model.found.length === 0) {
+      logError(`No graphql file found inside model.`)
+      process.exit(0)
+    }
+
+    const path = `${process.cwd()}/${model.found[0]}`
+    const schemaText = readFileSync(path, 'utf8')
+
+    const outputSchemaPath = `${process.cwd()}/generated/schema.graphql`
+    const outputResolverPath = `${process.cwd()}/generated/resolvers.ts`
+    
+    const backend = new GraphQLBackendCreator(schemaText)
+    
+    const connectionConfig = {
```

Comment:
WOW - what is that? and Why we have it here? 
Why we need to have hardcoded data for postgress connecting to localhost for generate comment (btw.. we will support more than postgress later.)

Are we no longer doing this split between generate and db commands?

Reply:
If that is just temporary then ok, but I would love to see it gone and refactor underlying generator to not require it.

Reply:
Actuallly db config is not needed here. I put them cuz I thought it was needed. My bad